src/compile/compile.ts

```diff
@@ -48,7 +48,7 @@ export function compileRootGroup(model: Model) {
   const spec = model.spec();
 
   let rootGroup:any = extend({
-      name: spec.name ? spec.name + '-root' : 'root',
+      name: spec.name ? spec.name + '-rootCell' : 'rootCell',
```

Comment:
I had forgotten about this myself. Perhaps the root mark in Vega should be named `scene` instead to correspond to the `scene` property we introduced recently. Hmm.